Source/Scene/Expression.js

```diff
@@ -917,19 +907,18 @@ define([
         }
 
         var member = this._right.evaluate(frameState, feature);
-        if (property instanceof Color) {
-            // Color components may be accessed with .x, .y, .z, .w and implicitly with .red, .green, .blue, .alpha
-            if (member === 'x') {
-                return property.red;
-            } else if (member === 'y') {
-                return property.green;
-            } else if (member === 'z') {
-                return property.blue;
-            } else if (member === 'w') {
-                return property.alpha;
+        if ((property instanceof Cartesian2) || (property instanceof Cartesian3) || (property instanceof Cartesian4)) {
+            // Vector components may be accessed with .red, .green, .blue, .alpha and implicitly with .x, .y, .z, .w
```

Comment:
Let's keep it, but go with `r`, `g`, `b`, `a`.  I went with the full words to start because I was copying the Cesium `Color` type, but we should stay consistent with GLSL so the styling language isn't too much of a hybrid.

Also, given that styles are often very concise and look like equations, this will fit well.

@pmconne this will be a small breaking change for your styles.